test/browserstack/options.mjs

```diff
@@ -136,6 +136,16 @@ export const operatingSystems = {
 		'El Capitan': osxResolutions,
 		'Sierra': osxResolutions,
 		'High Sierra': osxResolutions,
-		'Mojave': osxResolutions
+		'Mojave': osxResolutions,
+		'Catalina': osxResolutions,
+		'Big Sur': osxResolutions,
 	}
 };
+
+export const minVersions = {
+	"Chrome": "72.0",
+	"Opera": "58.0",
+	"Firefox": "65.0",
+	"Safari": "13.0",
+	"Edge": "44.0"
+};
```

Comment:
I'd add a comment that we're taking it from 2019 at https://en.wikipedia.org/wiki/Timeline_of_web_browsers#2010s